python_loops.py

```diff
@@ -0,0 +1,81 @@
+def your_name_list():
+    names_list = ['Jiri', 'Jan', 'Marie', 'Petr', 'Jana', 'Josef','Pavel', 'Martin', 'Jaroslav', 'Tomas', 'Eva', 'Miroslav','Hana', 'Anna', 'Zdenek', 'Frantisek', 'Vaclav', 'Michal','Lenka', 'Katerina']
+    my_name = input("Whats your name? ")
+    if my_name in names_list:
+        print(" Hey, that name is pretty ordinary. I mean... it's a cool name, but not that special.")
+    else: 
+        print(" Wow. Nice unique name, queen. ") #Replace with "king" or "monarch" if needed
+
+your_name_list()
+
+def loopy_fruit():
+    my_fruits = ['green apples', 'cherries', 'pineapple on pizza', 'peaches', 'plums', 'strawberries']
+    for fruit in my_fruits:
+        print("I like " + fruit)
+
+loopy_fruit()
+
+def spelling_alphabet():
+    #Řešení inspirováno: https://towardsdatascience.com/simple-morse-code-decoder-in-python-39f6db635af2
+    d = {'a':'alfa', 'b':'bravo', 'c':'charlie', 'd':'delta','e':'echo', 'f':'foxtrot', 'g':'golf', 'h':'hotel', 'i':'india','j':'juliett', 'k':'kilo', 'l':'lima', 'm':'mike','n':'november', 'o':'oscar', 'p':'papa', 'q':'quebec','r':'romeo', 's':'sierra', 't':'tango', 'u':'uniform','v':'victor', 'w':'whiskey', 'x':'x-ray', 'y':'yankee','z':'zulu'}
+    
+    usr_msg = list(input("Please insert your name. I will spell it for you "))
+    spelled_msg = []
+    for letter in range (0, len(usr_msg)):
+        if usr_msg[letter] in d.keys():
+            spelled_msg.append(d[usr_msg[letter]])#I dont get this. Like... at all. Maybe Im just tired, but I dont get dicts. 
+            #It almost works, but I honesly have no idea how. I'll look more into it on the weekend. 
+    print(spelled_msg)
+        
+
+
+spelling_alphabet()
+
+def shopping_list(): #The program in the excercise didnt make much sense to me. So I rewrite it in order to be useful. Now you can actually make your own shopping lists with it. 
```

Comment:
Klidne, jen dve poznamky:
1. Python ma radeji kratke radky, tak kdyz uz to pisete jako komentar pod funkci, pouzijte docstring.
2. Pozor na upravy funkci bez domluvy. U ukolu to samozrejme nevadi, ale nekdo dalsi muze spolehat na to, ze funkce funguje (byt divne) tak jak byla specifikovana a navazuje na to dalsi cinnost.